File: oldSnacks/Python/check_out.py
customer_name = input("What is the customer's name: ")
	
cashier_name = input("Enter cashier's name: ")

total = 0; 
vat = 0.075;

while True:
	user_purchased = input("What did the user buy?:")

	number_of_pieces = int(input("How many pieces:"))

	unit_price = float(input("How much per unit?:"))
	
	total = number_of_pieces * unit_price

	customer_response = input ("Add more items? (yes/no): ")
	if customer_response == "no": 
		break	
	
	amount_colected = ("How much did the customer gave you?: ")

	discount_percentage = input("Enter discount percentage (7 for 7%): ")
	discountpercentage = discount_percentage / 100
	discount = total * discount_percentage
	vat = total * vat
	total_sum = total - discount + vat
	total_amount = total + vat
	balance_recieved = amount_colected - total_amount 

print("SEMICOLON STORES")
print("MAIN BRANCH")
print("LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.")
print("TEL: 03293828343")
print("DATE : 18-AUGUST-24  8:48:11pms")
print(f"Cashier's name: {cashier_name}")
print("Customer Name: customer_name}")
print("\n=========================================================\n")
print(f"{'ITEM':>22}{'QTY':>10}{'PRICE':>10}{'TOTAL(NGN)':>15})")
print("\n---------------------------------------------------------\n")

print("---------------------------------------------------------")
print(f"Subtotal: {total:38.2f} NGN")
print(f"Discount: {discount:37.2f} NGN")
print(f"VAT (7.5%): {vat:36.2f} NGN")
print(f"Total Payable: {total_sum:30.2f} NGN")
	
print("=========================================================")
printf("Bill total: {total_amount:38.2f} NGN")
print(f"Amount paid: {total_sum:30.2f} NGN")
print(f"Balance: {balance_recieved:36.2f} NGN")

print("==========================================================")

print("THANK YOU FOR YOUR PATRONAGE!");

print("===========================================================");

File: oldSnacks/Java/CheckOut.java
import java.util.Scanner;
	public class CheckOut{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);

	System.out.println("What is the customer's name: ");
        String customerName = input.nextLine();
	
	System.out.println("Enter cashier's name: ");
        String cashierName = input.nextLine();


	double total = 0; 
	double vat = 0.075;

while (true){
	System.out.println("What did the user buy?:");
	String userPurchased = input.nextLine();

	System.out.println("How many pieces:");
        int numberOfPieces = input.nextInt();

	System.out.println("How much per unit?:");
	double unitPrice = input.nextDouble();
	
	total = numberOfPieces * UnitPrice;

	System.out.print("Add more items? (yes/no): ");
        input.nextLine(); 
        String customerResponse = input.nextLine();
            if (customerResponse .equalsIgnoreCase("no")) {
                break;
	    }
	}	
	
	System.out.println("How much did the customer gave you?:");
	int amountcolected = input.nextInt();

	System.out.println("Enter discount percentage (e.g., 10 for 10%): ");
        double discountpercentage = input.nextDouble();
	discountpercentage = discountpercentage / 100;
        double discount = total * discountpercentage;
       	vat = total * vat;
        double totalSum = total - discount + vat;
	double totalAmount = total + vat;
	double balanceRecieved = amountcolected - totalAmount; 

	
System.out.println("SEMICOLON STORES");
	System.out.println("MAIN BRANCH");
	System.out.println("LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.");
	System.out.println("TEL: 03293828343");
	System.out.println("DATE : 18-AUGUST-24  8:48:11pms");
	System.out.println("Cashier's name" + cashierName);
	System.out.println("Customer Name:" + customerName);
	System.out.println("\n=========================================================\n");
	System.out.printf("%22s%10s%10s%15s", "ITEM", "QTY", "PRICE", "TOTAL(NGN)");
	System.out.print("\n---------------------------------------------------------\n");

System.out.println("---------------------------------------------------------");
        System.out.printf("Subtotal: %38.2f NGN%n", total);
        System.out.printf("Discount: %37.2f NGN%n", discount);
        System.out.printf("VAT (7.5%%): %36.2f NGN%n", vat);
        System.out.printf("Total Payable: %30.2f NGN%n", totalSum);
	
System.out.println("=========================================================");
System.out.printf("Total amount: %38.2f NGN%n",totalAnount);
System.out.printf("Amount paid: %30.2f NGN%n", amountcolected);
System.out.printf("Balance: %36.2f NGN%n", balanceRecieved);

System.out.println("==========================================================");

System.out.println("THANK YOU FOR YOUR PATRONAGE!");

System.out.println("===========================================================");



	}	

}